src/providers/secondary-media-loader.ts

```diff
@@ -37,7 +38,8 @@ export class SecondaryMediaLoader implements ILoader {
       responseProfile: {
         type: INCLUDE_FIELDS,
         fields: 'id'
-      }
+      },
+      ks: params.ks
```

Comment:
the loader can't know it gets ks on param. move this assignment to the provider where it creates the loader 
see https://github.com/kaltura/playkit-js-providers/blob/master/src/k-provider/common/data-loader-manager.js#L55 - so something like:
`request.params.ks = parmas.ks`